### PR TITLE
New version zingo 1.2.2 123

### PR DESCRIPTION
**Pull request by juanky201271**
New production version of Zingo.

I used to build the Apps:
* zingolib -> commit: https://github.com/zingolabs/zingolib/commit/48c4b4f745dd1a6c5d1cf310df335ed5573f651c
* zingo-mobile -> commit: https://github.com/zingolabs/zingo-mobile/commit/e9a0223f0f3ccb38923d32d920c4cecbccfcdfff

What's new:
* Basic Finsight (Financial Insight)
* New option for Privacy (High / Normal)

@AloeareV @fluidvanadium @zancas @Oscar-Pepper

### CODE REVIEW

File: components/Send/Send.tsx [hunk 1]
```diff
@@ -75,7 +75,7 @@ const Send: React.FunctionComponent<SendProps> = ({
   const defaultFee = info.defaultFee || Utils.getFallbackDefaultFee();
   const { decimalSeparator } = getNumberFormatSettings();
   const spendable = totalBalance.transparentBal + totalBalance.spendablePrivate + totalBalance.spendableOrchard;
-  const stillConfirming = spendable !== totalBalance.total;
+  const stillConfirming = parseFloat(spendable.toFixed(8)) !== totalBalance.total;
```
Comment (Oscar-Pepper) [inline components/Send/Send.tsx:78]: i have not been able to find where `totalBalance.total` is fixed to 8 decimal places in the codebase but i can see in the app that is the case. i have tried to produce a manual test for the `stillConfirming` boolean but not entirely sure what im looking for... did you write a test for this or manually test that this works when you fix `spendable` to `8`? sorry im still getting used to the typescript codebase

**Review by Oscar-Pepper — COMMENTED** [review]
looks good, just want to check the change to `spendable` was tested or details on a test i can perform manually

File: components/Send/Send.tsx (hunk 1 above)
Comment (Oscar-Pepper) [reply components/Send/Send.tsx:78]: ok i have tested this and posted results in chat, could you have a look and explain the behaviour. thanks!

Comment (juanky201271) [reply components/Send/Send.tsx:78]: I saw very rarely a edge case in IOS that `spendable` here as a sum of these three fields can have more than 8 decimals with some number at the end, and in this cases, the spendable amount is red for no reason...

Something like this:
* spendable orchar -> 0.00012222
* spendable sapling -> 0.00013333
* transparent -> 0.00011111

spendable -> 0.000366660000000002

And here always the spendable is different to the total... and red for no reason. I need to investigate this really weird case, but doing this: `parseFloat(spendable.toFixed(8))` I'm 100% sure that the problem disappear.

Comment (Oscar-Pepper) [reply components/Send/Send.tsx:78]: ok great im approving now

**Review by Oscar-Pepper — APPROVED** [review]
wonderful!

**Pull request merged**